<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-JNGBS137" failures="0" tests="1" name="com.Excel_Read_1.copy_1.Excel_Read_1_1" time="0.567" errors="1" timestamp="2022-09-21T21:42:35 IST" skipped="0">
  <testcase classname="com.Excel_Read_1.copy_1.Excel_Read_1_1" name="getValues" time="0.567">
    <error message="
[public void com.Excel_Read_1.copy_1.Excel_Read_1_1.getValues(java.util.HashMap)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: getValues([Parameter{index=0, type=java.util.HashMap, declaredAnnotations=[]}])
Arguments: [(java.util.ArrayList) [{User 5=O, User 1=K, User 2=L, User 3=M, TESTCASE_ID=TC_1, User 4=N}, {User 5=T, User 1=P, User 2=Q, User 3=R, TESTCASE_ID=TC_2, User 4=S}, {User 5=E, User 1=A, User 2=B, User 3=C, TESTCASE_ID=TC_3, User 4=D}]]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.Excel_Read_1.copy_1.Excel_Read_1_1.getValues(java.util.HashMap)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: getValues([Parameter{index=0, type=java.util.HashMap, declaredAnnotations=[]}])
Arguments: [(java.util.ArrayList) [{User 5=O, User 1=K, User 2=L, User 3=M, TESTCASE_ID=TC_1, User 4=N}, {User 5=T, User 1=P, User 2=Q, User 3=R, TESTCASE_ID=TC_2, User 4=S}, {User 5=E, User 1=A, User 2=B, User 3=C, TESTCASE_ID=TC_3, User 4=D}]]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- getValues -->
  <system-out/>
</testsuite> <!-- com.Excel_Read_1.copy_1.Excel_Read_1_1 -->
